src/main/scala/Exercise.scala

```diff
@@ -0,0 +1,22 @@
+class Exercise {
+  def FirstSolution(){
+    val arrayRange = (1 to 99).toArray[Int]
+    arrayRange.reverse.foreach(arrayRange => println(s"\n$arrayRange bottle of jojo on the wall\n$arrayRange bottle of jojo on the wall\ntake one down pass it around ${arrayRange - 1} bottles of jojo in the wall"))
+  }
+  def SecondSolution(beerNum:Int){
+    println(s"\n$beerNum bottle of JOJO on the wall\n$beerNum bottle of JOJO on the wall")
+    if(beerNum == 0){
+        println("NO MORE")
+    }else SecondSolution(beerNum - 1)
+  }
+  def DupArray(){
```

Comment:
method names start with a lowercase word, only words after should have capitalised first letter